lib/public/js/chart.js

```diff
@@ -149,7 +140,14 @@ const toggleColor = () => {
     // eslint-disable-next-line no-undef
     singleColor = chart({ limit: 20, singleColor: !singleColor });
     const btn = document.getElementById('btn-toggle');
-
     btn.classList.toggle('active');
     btn.classList.toggle('inactive');
 };
+
+(typeof module !== 'undefined' && module !== null ? module : {}).exports = {
```

Comment:
can't we just do `module.exports` ?

Reply:
using module.exports gave an error in the console while running the application. `module is not defined`.